lms/resources/_js_config/file_picker_config.py

```diff
@@ -4,6 +4,8 @@ class FilePickerConfig:
     @classmethod
     def blackboard_config(cls, context, request, application_instance):
         """Get Blackboard files config."""
+        if not application_instance.product == application_instance.Product.BLACKBOARD:
+            return {}
```

Comment:
I don't think we want this, for the same reasons as given in https://github.com/hypothesis/lms/pull/3461

Reply:
An alternative could be something along this lines:

```diff
diff --git a/lms/resources/_js_config/file_picker_config.py b/lms/resources/_js_config/file_picker_config.py
index 46864e7e..a92ffba8 100644
--- a/lms/resources/_js_config/file_picker_config.py
+++ b/lms/resources/_js_config/file_picker_config.py
@@ -8,11 +8,14 @@ class FilePickerConfig:
             return {}
 
         enabled = application_instance.settings.get("blackboard", "files_enabled")
+        groups_enabled = application_instance.settings.get(
+            "blackboard", "files_enabled"
+        )
 
         auth_url = request.route_url("blackboard_api.oauth.authorize")
         course_id = request.params.get("context_id")
 
-        return {
+        config = {
             "enabled": enabled,
             "groupsEnabled": context.blackboard_groups_enabled,
             "listFiles": {
@@ -21,13 +24,17 @@ class FilePickerConfig:
                     "blackboard_api.courses.files.list", course_id=course_id
                 ),
             },
-            "listGroupSets": {
+        }
+
+        if groups_enabled:
+            config["listGroupSets"] = {
                 "authUrl": auth_url,
                 "path": request.route_path(
                     "blackboard_api.courses.group_sets.list", course_id=course_id
                 ),
-            },
-        }
+            }
+
+        return config
 
     @classmethod
     def canvas_config(cls, context, request, application_instance):
```